src/main/java/org/alliancegenome/curation_api/services/validation/dto/fms/GeneExpressionAnnotationFmsDTOValidator.java

```diff
@@ -0,0 +1,94 @@
+package org.alliancegenome.curation_api.services.validation.dto.fms;
```

Comment:
I'd like this to be split out into the hierarchy of ExpressionAnnotationFmsDTOValidator and GeneExpressionAnnotationFmsDTOValidator